vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+// autoload.php @generated by Composer
+
+require_once __DIR__ . '/composer/autoload_real.php';
+
+return ComposerAutoloaderIniteb89c64049e744028a24868d3fed306f::getLoader();
```

Comment:
You do not need to include vendor folder for native modules. It will be generated at the release.

Reply:
Hmmz I copied this approach from other modules. Wasn't aware of your build process. Will remove it with the next iteration.